pkgs/applications/networking/nextcloud-client/default.nix

```diff
@@ -15,20 +15,22 @@
 , qtquickcontrols2
 , qtgraphicaleffects
 , sqlite
+, qtwebsockets
 }:
 
 mkDerivation rec {
   pname = "nextcloud-client";
-  version = "3.0.3";
+  version = "3.1.3";
 
   src = fetchFromGitHub {
     owner = "nextcloud";
     repo = "desktop";
     rev = "v${version}";
-    sha256 = "0idh8i71jivdjjs2y62l22yl3qxwgcr0hf53dad587bzgkkkr223";
+    sha256 = "sha256-8Ql6tOvWOjAvMJA87WlT9TbpnbciBsjDxRuYlMVi/m8=";
```

Comment:
I didn't know that sha256's can have this format.

Reply:
https://github.com/NixOS/nix/commit/6024dc1d97212130c19d3ff5ce6b1d102837eee6